CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+# [v1.3.8] - 2019-12-20
```

Comment:
```suggestion
## [v1.3.8] - 2019-12-20
```

Reply:
KeepAChangelog formatting requires this be a double pound sign

Reply:
Good catch. Fixed.